server/src/controllers/download.controller.ts

```diff
@@ -26,6 +27,18 @@ export class DownloadController {
     return this.service.getDownloadInfo(auth, dto);
   }
 
+  @Post('request')
+  @Authenticated({ permission: Permission.AssetDownload, sharedLink: true })
+  @Endpoint({
+    summary: 'Prepare download archive',
+    description:
+      'Create a download request for the specified assets or album. The response includes one or more tokens that can be used to download groups of assets.',
+    history: new HistoryBuilder().added('v2').stable('v2'),
+  })
+  prepareDownload(@Auth() auth: AuthDto, @Body() dto: DownloadInfoDto): Promise<PrepareDownloadResponseDto> {
+    return this.service.prepareDownload(auth, dto);
+  }
```

Comment:
I'm not fond of the endpoint URL nor the descriptions, so if anyone has suggestions I'd appreciate them